python_modules/dagster/dagster/_core/definitions/selector.py

```diff
@@ -187,6 +187,36 @@ def from_graphql_input(graphql_data):
         )
 
 
+class ResourceSelector(
```

Comment:
Should we just start using python3-style NamedTuples right now?

```python
class ResourceSelector(NamedTuple):
   location_name: str
   repository_name: str
   resource_name: str
```

Reply:
sure, updated